pkgs/by-name/bc/bcu/package.nix

```
@@ -0,0 +1,50 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  libftdi1,
+  libusb1,
+  libyaml,
+  ncurses,
+  nix-update-script,
+  pkg-config,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bcu";
+  version = "1.1.119";
+
+  src = fetchFromGitHub {
+    owner = "nxp-imx";
+    repo = "bcu";
+    tag = "bcu_${finalAttrs.version}";
+    hash = "sha256-GVnUkIoqHED/9c3Tr4M29DB+t6Q8OPDcxVWKNn/lU/8=";
+  };
+
+  enableParallelBuilding = true;
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    libftdi1
+    libusb1
+    libyaml
+    ncurses
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "NXP i.MX remote control and power measurement tools";
+    homepage = "https://github.com/nxp-imx/bcu";
+    license = lib.licenses.bsd3;
+    mainProgram = "bcu";
+    maintainers = [ lib.maintainers.jmbaur ];
+    platforms = lib.platforms.linux;
```

The upstream seems to also support Mac
```suggestion
    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

---

I needed some other fixes to make this work, but it now builds for macos